cmd/containerboot/main.go

```diff
@@ -651,6 +651,8 @@ func tailscaleUp(ctx context.Context, cfg *settings) error {
 	}
 	if cfg.Routes != "" {
```

Comment:
Or remove the conditional and leave a comment here and below

Reply:
yeah that's better